### PR TITLE
page plan dev

### PR DESCRIPTION
**Pull request by bzkrouse**
@statasaurus I'll do the json & vignette in a separate PR!

### CODE REVIEW

File: R/apply_tfrmt.R [hunk 1]
```diff
@@ -72,41 +56,119 @@ apply_tfrmt <- function(.data, tfrmt, mock = FALSE){
...
+#' @param col_plan_vars col plan converted to vars() to be applied
+#' @param big_n_df tbl of big Ns to apply (if 1 set of overall Ns) or list of tbl of bigNs (if grouping by page)
+#' to apply
+#'
+#' @importFrom purrr map map2
+#' @noRd
+apply_tfrmt_subtable_mapper <- function(tfrmt, .data, col_plan_vars, big_n_df){
+
+  if (inherits(.data, "list") && length(.data)>1){
+    if (inherits(.data, "list") && inherits(big_n_df, "list") && length(big_n_df)>1){
```
Comment (statasaurus) [inline R/apply_tfrmt.R:99]: Can you add a comment this is for if there are multiple different big_n headings

File: R/print_to_gt.R [hunk 2]
```diff
@@ -213,11 +239,17 @@ cleaned_data_to_gt <- function(.data, tfrmt){
...
+      stub_row_group.border.width = px(0),
       stub_row_group.border.color = "transparent",
+      row_group.border.bottom.width = px(0),
       row_group.border.bottom.color = "transparent",
       row_group.border.top.color = "transparent",
-      table.font.names = c("Courier", default_fonts())) %>%
+      table.font.names = c("Courier", default_fonts()),
+      page.numbering = TRUE,
+      page.header.use_tbl_headings = FALSE,
+      page.orientation = "landscape") %>%
```
Comment (statasaurus) [inline R/print_to_gt.R:252]: I am glad that worked!!

**Review by statasaurus — APPROVED** [review]
This looks great! I would just add one comment really

File: R/apply_page_plan.R [hunk 3]
```diff
@@ -0,0 +1,440 @@
...
+#' @importFrom rlang is_empty
+apply_page_plan <- function(.data, page_plan, group, label, row_grp_plan_label_loc = "indented"){
+
+  # preferentially apply any page_structures;
+  # if none, then apply max_rows
+  if (!is_empty(page_plan$struct_list)){
+    apply_page_struct(.data, page_plan$struct_list, group, label, page_plan$note_loc)
+  } else if (!is.null(page_plan$max_rows)){
+    apply_page_max_rows(.data, page_plan$max_rows, group, label, row_grp_plan_label_loc)
+  } else {
```
[the remark this answers is not in the document]
Comment (bzkrouse) [reply R/apply_page_plan.R:22]: there's a message now, thanks for the suggestion!

**Pull request merged**